src/components/v5/shared/Stepper/types.ts

```diff
@@ -9,6 +9,7 @@ export interface StepperItem<TKey> {
   isOptional?: boolean;
   isSkipped?: boolean;
   isHidden?: boolean;
+  iconName?: string;
```

Comment:
It shouldn't be added here. It should already exist inside `StepperButtonProps` interface